Libraries/Vibration/VibrationIOS.android.js

```diff
@@ -6,18 +6,28 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * Stub of VibrationIOS for Android.
- *
- * @providesModule VibrationIOS
+ * @providesModule VibrationAndroid
  */
+
 'use strict';
 
-var warning = require('warning');
+var RCTVibrationAndroid = require('NativeModules').VibrationAndroid;
+
+/**
+ * This exposes the native VibrationAndroid module as a JS module. This has a function 'vibrate'
+ * which takes the following parameters:
+ *
+ * 1. int duration: The duration of the vibration in milliseconds.
+ */
+
+var VibrationAndroid = {
+
+  vibrate: function (
+    duration: number
```

Comment:
[VibrationIOS.vibrate](https://github.com/facebook/react-native/blob/master/Libraries/Vibration/VibrationIOS.ios.js) doesn't take an argument so you need to make sure this works also when no argument is passed so that we are consistent between iOS and Android.